app/src/Util.re

```diff
@@ -0,0 +1,12 @@
+module CssHelper = {
+  open Css;
+  let (<<) = (a, b) => merge([a, b]);
```

Comment:
Cute little operator!

Generally I have an inert resistance towards packages/modules that are called common, util, shared, etc. On the surface the naming is not strong indication for the functionality it provides as everything in a project should serve some form of utility. It opens up a catch-all bucket for everything that hasn't been placed in the right place.

If we need operators for our styles, let's put them in an equally named module.

Reply:
Good point.

How about putting everything design system related in a `DesignSystem.re` file? I feel like having a separate module called `Styles` could be confusing, because our components and pages have modules in them which are already called `Styles`:
```
module Styles = {
  open Css;
  open Particle;

  let regular =
    merge([style([color(Color.darkGray())]), style(Particle.Font.text)]);
};

[@react.component]
let make = (~children) => <p className=Styles.regular> children </p>;
```

What do you think?

Reply:
Makes perfect sense. `open DesignSystem` or `open DesignSystem.Operators` reads nicely.